portal/server/routes/contract/bond/index.js

```diff
@@ -38,24 +38,14 @@ const userCanAccessBond = (user, deal) => {
 
   const { status } = deal.details;
 
-  if (status === 'Ready for checker\'s approval'
+  return !(status === 'Ready for checker\'s approval'
```

Comment:
should we add this inside an array? there are a few conditions here
i.e.
```
const array = ['Ready for checker\'s approval', 'Acknowledged', etc]
```
these items ^ could come from CONSTANTS as well
```
if (!array.includes(status)) {...}
```

Reply:
Good spot, have improved the code quality.